content/lang/tr/guide/differences-to-react.md

```diff
@@ -3,80 +3,75 @@ name: Differences to React
 permalink: '/guide/differences-to-react'
 ---
 
-# Differences to React
+# React’tan Farklılıkları
 
-Preact itself is not intended to be a reimplementation of React.  There are differences.  Many of these differences are trivial, or can be completely removed by using [preact-compat], which is an thin layer over Preact that attempts to achieve 100% compatibility with React.
+Preact’ın kendisi React’ın yeniden tasarlanmış hali olmayı amaçlamamıştır. Aralarında farklılıklar vardır. Bu farklılıkların bir çoğu önemsiz yada [preact-compat]kullanılarak tamamen kaldırılabilir. Preact-compat ise preact üzerin bulunan, react ile %100 uyumluluk ile çalışan bir katmandır. İşlevi react’a çalışan modullerin preact’ta uyumlu olarak çalışmasını sağlamaktır.
 
-The reason Preact does not attempt to include every single feature of React is in order to remain **small** and **focused** - otherwise it would make more sense to simply submit optimizations to the React project, which is already a very complex and well-architected codebase.
+Preact’ın React’ın her bir özelliğini içermemesinin nedeni, **odaklanmış** ve **küçük** kalmak istemesidir. Aksi takdirde çok karmaşık ve iyi tasarlanmış kod tabanına sahip olan React projesinde iyileştirmeler yapmak çok daha mantıklı olur.
 
+## Sürüm Uyumluluğu
 
-## Version Compatibility
+Hem preact hem [preact-compat] için versiyon uyuyumluğu, React’ın mevcut ve önceki ana sürümlerine göre ölçülür. React takımın tarafından yeni özellikler duyurulduğu zaman, [proje hedefleri] göz önüne alınarak mantıklı bulunan özellikler Preact’ın çekirdeğine eklenir. Bu, sürekli tartışmalar ve açık kararlar vasıtasıyla, sorunları ve pull requestleri kullanarak geliştirilen oldukça demokratik bir süreçtir.
 
-For both Preact and [preact-compat], version compatibility is measured against the _current_ and _previous_ major releases of React. When new features are announced by the React team, they may be added to Preact's core if it makes sense given the [Project Goals].  This is a fairly democratic process, constantly evolving through discussion and decisions made in the open, using issues and pull requests.
+> Dolayısıyla, web sitesi ve doküman uyumluğu tartışılırken yada karşılaştırma yapılırken React `0.14.x` ve `15.x`‘i yansıtır.
 
-> Thus, the website and documentation reflect React `0.14.x` and `15.x` when discussing compatibility or making comparisons.
+## Neler Dahil Edilir?
 
+*   [ES6 Class Components]
+    *   _sınıflar, durum bilgisi olan bileşenleri tanımlamanın anlamlı yolunu sunar_
```

Comment:
```
- _`class`'lar, state tutan olan component'leri tanımlamak için tanımlayıcı bir yol sunar_
```

Reply:
desek cok mu ing tabir oluyor icinde?

Reply:
`class`, `component` ve `state` temel olduğu için çevirmem mantıksız olmuş.